frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -58,12 +58,30 @@ export const getAssociatedNodes = (pvs: K8sResourceKind[]): string[] => {
   return Array.from(nodes);
 };
 
+export const shouldDeployInternalAsMinimal = (nodes: NodeKind[]) => {
+  const { totalCPU, totalMemory } = nodes.reduce(
+    (acc, curr) => {
+      const cpus = humanizeCpuCores(getNodeCPUCapacity(curr)).value;
+      const memoryRaw = getNodeAllocatableMemory(curr);
+      const memory = humanizeBinaryBytes(convertToBaseValue(memoryRaw)).value;
+      acc.totalCPU += cpus;
+      acc.totalMemory += memory;
+      return acc;
+    },
+    {
+      totalCPU: 0,
+      totalMemory: 0,
+    },
+  );
+  return totalCPU < 30 || totalMemory < 60;
+};
+
 export const shouldDeployMinimally = (nodes: NodeKind[]) =>
```

Comment:
```suggestion
export const DeployBaremetalAsMinimal = (nodes: NodeKind[]) =>
```

Reply:
You can deploy Attached devices in AWS as well.

Reply:
I will keep it as `shouldDeployAttachedAsMinimal` to keep it consistent with the other function